pyface/ui/wx/action/action_item.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, parent, menu, item, controller):
         action  = item.action
         label   = action.name
         kind    = _STYLE_TO_KIND_MAP[action.style]
-        longtip = action.description
+        longtip = action.description or action.tooltip
```

Comment:
Sure, that's fine; it just needs to be propagated to the `qt4` version of this code as well.